code/Language/Drasil/ExprExtract.hs

```diff
@@ -0,0 +1,50 @@
+module Language.Drasil.ExprExtract (egetDoc)where
```

Comment:
This should be called `Language.Drasil.Document.Extract`.

Reply:
Changes were made in 8c1738588678f66ed29a6da31c4b7a6f243eaabe.